core/store/presenters/presenters.go

```diff
@@ -83,7 +84,11 @@ func showBalanceForAccount(store *store.Store, account accounts.Account, balance
 func getBalance(store *store.Store, account accounts.Account, balanceType requestType) (balanceable, error) {
 	switch balanceType {
 	case ethRequest:
-		return store.TxManager.GetEthBalance(account.Address)
+		bal, err := store.EthClient.BalanceAt(context.TODO(), account.Address, nil)
```

Comment:
Another context.TODO. pls grep codebase for these

Reply:
This is intentional.  We have another story to utilize contexts everywhere, and the purpose of context.TODO is to allow for gradual migration from code that doesn't use them to code that does.